bundle/edu.gemini.pot/src/main/java/edu/gemini/spModel/target/SPTarget.java

```diff
@@ -75,40 +72,4 @@ public SPTarget clone() {
         return new SPTarget(_target.clone());
     }
 
-
-    ///
-    /// END OF PUBLIC API ... EVERYTHING FROM HERE DOWN GOES AWAY
-    ///
-
-    /** Get the PM radial velocity in km/s if the contained target is sidereal, otherwise zero. */
-    public double getTrackingRadialVelocity() {
-        double res = 0.0;
-        if (_target instanceof HmsDegTarget) {
-            final HmsDegTarget t = (HmsDegTarget)_target;
-            res = t.getRV().getValue();
-        }
-        return res;
-    }
-
-    /** Set the PM radial velocity in km/s if the contained target is sidereal, otherwise throw. */
-    public void setTrackingRadialVelocity(final double newValue) {
-        if (_target instanceof HmsDegTarget) {
-            final HmsDegTarget t = (HmsDegTarget)_target;
-            final RV rv = new RV(newValue);
-            t.setRV(rv);
-            _notifyOfUpdate();
-        } else {
-            throw new IllegalArgumentException();
-        }
-    }
-
-    // I'm making this public so I can call it from an editor when I make
-    // a change to the contained target, rather than publishing all the
-    // target members through this idiotic class. All of this crap needs
-    // to be rewritten.
-    /** @deprecated */
-    public void notifyOfGenericUpdate() {
```

Comment:
They're necessary if you change the `ITarget` **directly** because these changes won't generate a property change event on the `SPTarget`. But now that all the mutators are on `SPTarget` I think we can probably get rid of `notifyOfGenericUpdate()` … I'll check and see.